src/jit/lclvars.cpp

```diff
@@ -4100,7 +4100,16 @@ void Compiler::lvaComputeRefCounts(bool isRecompute, bool setSlotNumbers)
             // to do something else to ensure late temps are considered.
             for (lclNum = 0, varDsc = lvaTable; lclNum < lvaCount; lclNum++, varDsc++)
             {
-                assert(varDsc->lvImplicitlyReferenced);
+                const bool isSpecialVarargsParam = varDsc->lvIsParam && raIsVarargsStackArg(lclNum);
```

Comment:
raIsVarargsStackArg is under an x86 ifdef, I am worried this special case may not be correctly marked on other platforms.

Reply:
For example on x64 if there are 5 args the last arg would not be correctly marked isSpecialVarargsParam correct?